res/values/strings.xml

```diff
@@ -785,6 +815,10 @@
     <string name="menu.view.developer.open.log.folder">Open the log folder</string>
     <string name="menu.view.developer.open.current.log.file">Open current logfile</string>
     <string name="user_location_permission_explanation">Delta Chat needs the location permission in order to show and share your location.</string>
+    <string name="ask_delete_message_desktop">Are you sure you want to delete this message?</string>
+    <string name="explain_desktop_minimized_disabled_tray_pref">Tray icon cannot be disabled as Delta Chat was started with the --minimized option</string>
+    <string name="no_spellcheck_suggestions_found">No Spelling Suggestions found</string>
+    <string name="mac_show_windows">Show Window</string>
```

Comment:
"Show windows" or "Show window"?

as the string is pretty neutral, i'd also remove the "mac_" prefix.

Reply:
lets keep the case here as this is shown in the mac titlebar and the style there is each word uppercase